src/fides/api/db/seed.py

```diff
@@ -401,7 +401,7 @@ async def load_default_taxonomy(async_session: AsyncSession) -> None:
             await upsert_resources(
                 sql_model_map[resource_type], resources, async_session
             )
-        except QueryError:  # pragma: no cover
+        except QueryError as e:  # pragma: no cover
```

Comment:
were you going to make use of `e` or can we remove? i'm going to remove for now and feel free to re-add, just because it's throwing a pylint error.

Reply:
removed in 9c4fe119b49793f9919a5e1d1361bc74ce8618e1